extensions/amp-viafoura/0.1/amp-viafoura.js

```diff
@@ -26,7 +28,7 @@ class AmpViafoura extends AMP.BaseElement {
   /** @override */
   preconnectCallback(onLayout) {
     // The viafoura iframe
-    this.preconnect.url('https://???.viafoura.com', onLayout);
+    this.preconnect.url(`https://${SRC_DOMAIN}`, onLayout);
```

Comment:
it looks like we won't need this since we're going to host the frame content at api.viafoura.com for now